### DISCUSSION TITLE
How to rename components scope?

### DISCUSSION THREAD

**Discussion by ivansky** [teambit/bit#4630] [Help] [answered]
Hi,

We have a bitdev account with the organisation name the same as component scope. 
Could we somehow rename components scope and leave the same organisation name in a bitdev account?
It already has a bunch of components, should we redeploy them all after renaming?

Thanks in advance!

**Comment by GiladShoham** [comment, marked as answer]
there is no rename scope, as the scope is part of the component id which means it's part of the module name.
So changing the scope name require you to change the import statement in all of the code.
Also, the import statements between the components are in the same scope.
Usually, the best practice is to deprecate the component with the old name and export it as a new component to another scope.
In your case where you want to rename the entire scope, I would import all the components into the same workspace (or if you already have a workspace contain them all), then use the `bit init --reset-new` to make all of them being new components.
Then change the default scope, all the import statements, and export them to the new scope.
Later after making sure it's working, I would just delete the entire old scope in case no one uses it from another scope.
In case you do use them from external scope, I would then deprecate all the components of the old scope.
